packages/e2e/features/ui/components/authenticator/custom-slots.feature

```diff
@@ -1,4 +1,4 @@
-Feature: Sign Up with Attributes
+Feature: Custom Slots 
```

Comment:
For some reason this name was set to `sign up with attributes` which was already on the list.